src/download.tsx

```diff
@@ -73,7 +74,8 @@ class Download extends KalturaPlayer.core.BasePlugin {
 
   private injectOverlayComponents() {
     this.iconId = this.upperBarManager.add({
-      label: 'Download',
+      //@ts-ignore
+      label: <Text id="download.download">Download</Text>,
```

Comment:
I saw lint check fails because of `@ts-ignore`
instead of @ts-ignore you can manually specify type:
`(<Text id="download.download">Download</Text>) as string`
or
`(<Text id="download.download">Download</Text>) as any`